mern/client/src/Interactor/recoverSessionInteractor.js

```diff
@@ -0,0 +1,18 @@
+import { getFlowBySessionID } from "../DAO/recoverSessionDAO"
+//  import dao to call axios function
+// manipulate data
+
+// sessionID does not need to be manipulated so this function sends it straight to DAO
+export async function sendToDAO(sessionID) {
+    // returns a flow promise object
+    const flow = await getFlowBySessionID(sessionID);
+    console.log("this is the flow", flow);
+    return flow;
+
+}
+
+// returns first QA pair from given flow promise object
+export function getQAFromFlow(flowPromise) {
```

Comment:
What is the purpose of this function, I don't see why you would ever need to fetch only the "id" for the starting 5 questions.

Reply:
This function is meant to return the current_question QA object from the flow, to display on the starting-intent screen. I will update it now that the flow's been updated.